src/components/input/Input.jsx

```diff
@@ -1,76 +1,89 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styles from './inputstyles.css';
 
 const Input = ({ url, body, onChange, onSubmit }) => (
```

Comment:
Input probably isn't the best name for a component because it's easy to confuse it with lowercase input